dev-db/percona-xtrabackup/percona-xtrabackup-2.4.14.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic
+
+DESCRIPTION="Hot backup utility for MySQL based servers"
+HOMEPAGE="https://www.percona.com/software/mysql-database/percona-xtrabackup"
+SRC_URI="https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-${PV}/source/tarball/${P}.tar.gz
+	mirror://sourceforge/boost/boost_1_59_0.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	app-arch/lz4:0=
+	app-editors/vim-core
+	dev-libs/libaio
+	dev-libs/libedit
+	dev-libs/libev
+	dev-libs/libevent:0=
+	dev-libs/libgcrypt:0=
+	dev-libs/libgpg-error
+	dev-python/sphinx
+	net-misc/curl
+	sys-libs/zlib:="
+
+RDEPEND="
+	${DEPEND}
+	!dev-db/percona-xtrabackup-bin
+	dev-perl/DBD-mysql"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-2.4.6-remove-boost-version-check.patch
+	"${FILESDIR}"/${PN}-2.4.11-fix-gcc6-isystem.patch
+)
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	# remove bundled boost, libedit, libevent, zlib
```

Comment:
There's no boost below.

Reply:
Thanks, removed. It's a relic from the past when we removed the bundled boost.

Reply:
Would be nice to restore that relic ;-P. I mean, removing bundled boost.

Reply:
Actually we had it like that before, but since mysql upstream is locked to a specific boost version, I don't think it's gonna return (we don't have those versions in portage anyways):

https://github.com/gentoo/gentoo/commit/9cdf4c1ca5d6b2a95a8918136132bc4d763229b8#diff-d5871f7b935c91fe61edf590ed06e01e

@Whissi and @SoapGentoo - what do you think?

Reply:
@mgorny Mysql project gave up moved to bundled boost lib usage.